compiler-core/src/type_/error.rs

```diff
@@ -958,6 +959,7 @@ pub fn convert_get_value_constructor_error(
             location,
             name,
             imported_modules,
+            hint: None,
```

Comment:
There should be no error formatting in the analysis modules, instead errors should be represented using types and then they are printed in the error module.